fixieai/client/client.py

```diff
@@ -166,3 +168,28 @@ def create_session(self) -> Session:
     def get_session(self, session_id: str) -> Session:
         """Return an existing Session object."""
         return Session(self, session_id)
+
+    def get_current_username(self) -> str:
+        """Returns the username of the current user."""
+        query = gql(
+            """
+            query getUsername {
+                user {
+                    username
+                }
+            }
+        """
+        )
+        result = self._gqlclient.execute(query)
+        assert "user" in result and isinstance(result["user"], dict)
+        user = result["user"]
+        assert "username" in user and isinstance(user["username"], str)
+        return user["username"]
+
+    def refresh_agent(self, agent_handle: str):
+        """Indicates that an agent's prompts should be refreshed."""
+        username = self.get_current_username()
+        requests.post(
+            f"{self._api_url}/api/refresh/{username}/{agent_handle}",
```

Comment:
optional nit: I tend to think we should get all paths from `constants.py` instead of constructing them here. This `FixieClient` probably should not get the `api_url` in the constructor and follow the pattern elsewhere that all paths come directly from `constants.py`.

Reply:
Some more context: We have this [`UrlHub`](https://github.com/fixie-ai/fixie/blob/e869647a84604385f9ba4225e9cf47db8049dcb3/src/backend/fixie/chain/api/agent.py#L109) object on the platform that we intend to pass to the 3P agents that introduces all service urls at some point. Not too relevant here but just FYI.

Reply:
Ah, yes I agree it's confusing that some places might be using different API URLs -- I removed the argument from the constructor to align this across the SDK.